lib/libutee/include/pta_widevine.h

```diff
@@ -0,0 +1,27 @@
+/* SPDX-License-Identifier: BSD-2-Clause */
+/*
+ * Copyright (c) 2023, The ChromiumOS Authors
+ */
+
+#ifndef __PTA_WIDEVINE_H
+#define __PTA_WIDEVINE_H
+
+/*
+ * Interface to the widevine pseudo-TA, which is used for passing parameters
+ * for widevine TA.
+ */
+#define PTA_WIDEVINE_UUID                                              \
+	{                                                              \
+		0x721f4da9, 0xda05, 0x40d4,                            \
+		{                                                      \
+			0xa1, 0xa3, 0x83, 0x77, 0xc1, 0xe0, 0x8b, 0x0a \
+		}                                                      \
+	}
+
+/*
+ * Widevine PTA supported commands
+ */
```

Comment:
please add a description of the commands. Something like:
```c
/*
 * PTA_WIDEVINE_GET_TPM_PUBKEY - Get Widevine TPM public key
 * PTA_WIDEVINE_GET_WIDEVINE_PRIVKEY - Get Widevine private key
 *
 * [out]     memref[0]        Retrieved key data
 *
 * Return codes:
 * TEE_SUCCESS - Invoke command success
 * TEE_ERROR_BAD_PARAMETERS - Incorrect parameters
 * TEE_ERROR_SHORT_BUFFER - Output buffer size is too small
 * TEE_ERROR_something - Requested data not available
 */
```
Note that describing the return code is not mandatory as long as it complies with the definition provided by the GP TEE specification.

I think the API should also describe the type/format of the keys, like what's done in _core/pta/widevine.c_. Not a strong opinion if you think it is implicitly state by keywords _"Widevine TPM public key"_ and _"Widevine private key"_.

I think you need a error code for when the requested data is not available (for when `init_widevine_dt_data()` failed).